cluster_experiments/experiment_analysis.py

```diff
@@ -301,6 +301,92 @@ def from_config(cls, config):
         )
 
 
+class PairedTTestClusteredAnalysis(ExperimentAnalysis):
+    """
+    Class to run paired T-test analysis on aggregated data
+
+    Arguments:
+        cluster_cols: list of columns to use as clusters
+        target_col: name of the column containing the variable to measure
+        treatment_col: name of the column containing the treatment variable
+        treatment: name of the treatment to use as the treated group
+        strata: column to generate 2 groups (a and b)
+
+    Usage:
+
+    ```python
+    from cluster_experiments.experiment_analysis import TTestClusteredAnalysis
```

Comment:
you're not importing paired ttest (I think the unit tests will fail because of this)